Api/Program.cs

```diff
@@ -1,17 +1,29 @@
+using Npgsql;
 using System.Data;
-
 using Api.Application;
-
+using Api.Domain.Services;
+using Api.Domain;
 using FluentValidation.AspNetCore;
 
 var builder = WebApplication.CreateBuilder(args);
 
+var connectionString = builder.Configuration.GetConnectionString("DefaultConnection") ??
+                       "Host=localhost;Database=expense_tracker_group_5_postgres;Username=user;Password=password";
+
+builder.Services.AddTransient<IDbConnection>(sp => 
+    new NpgsqlConnection(connectionString));
+
 builder.Services.AddAuthorization();
 builder.Services.AddAuthentication(JwtConfiguration.ConfigAuthentication)
                 .AddJwtBearer(JwtConfiguration.ConfigBearer);
+
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen(SwaggerConfiguration.ConfigSwaggerGen);
+
 builder.Services.AddAutoMapper(typeof(Program));
+
+builder.Services.AddScoped<IIncomeService, IncomeService>();
```

Comment:
inject this dependencies using the DependencyInjector `Api/Application/Configurations/Dependencies/DependencyInjector.cs`

Reply:
Okey I try to do that